openslides/utils/autoupdate.py

```diff
@@ -64,7 +66,7 @@ def format_for_autoupdate(collection_string, id, action, data=None):
 
 
 @channel_session_user_from_http
-def ws_add_site(message):
+def ws_add_site(message: Any) -> None:
```

Comment:
Message is a specific type, isn't it?

Reply:
It is a Dict. Maybe you can use Dict[str, Any]. I don't belief that we can set an explicit type for the Value.